app/layout.tsx

```diff
@@ -11,8 +11,8 @@ const jetBrainsMono = JetBrains_Mono({
 })
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "TehcCringe",
+  description: "Artificial news generated by humans",
```

Comment:
Let's use the "official" description

```suggestion
  description: "Open source satirical tech news outlet",
```